packages/apollo/package.json

```diff
@@ -35,11 +35,11 @@
     "react": "^16.0.0 || ^17.0.0 || ^18.0.0"
   },
   "dependencies": {
-    "@uplift-ltd/constants": "^1.8.7",
-    "@uplift-ltd/sentry": "^4.1.5"
+    "@uplift-ltd/constants": "^1.8.7"
   },
   "devDependencies": {
     "@apollo/client": "^3.7.6",
+    "@sentry/types": "^7.5.0",
```

Comment:
Should this match the `^7.47.0` in the sentry package? Or they are not in sync?

Reply:
dang, good catch. twas a typo